### ISSUE TITLE
how to get recall50, recall75 log in training?

### ISSUE THREAD

**Issue by dndmania** [experiencor/keras-yolo3#192]
hello i have some question 
loss = tf.Print(loss, [grid_h, avg_obj], message='avg_obj \t\t', summarize=1000)
            loss = tf.Print(loss, [grid_h, avg_noobj], message='avg_noobj \t\t', summarize=1000)
            loss = tf.Print(loss, [grid_h, avg_iou], message='avg_iou \t\t', summarize=1000)
            loss = tf.Print(loss, [grid_h, avg_cat], message='avg_cat \t\t', summarize=1000)
            loss = tf.Print(loss, [grid_h, recall50], message='recall50 \t', summarize=1000)
            loss = tf.Print(loss, [grid_h, recall75], message='recall75 \t', summarize=1000)   
            loss = tf.Print(loss, [grid_h, count], message='count \t', summarize=1000)     
            loss = tf.Print(loss, [grid_h, tf.reduce_sum(loss_xy), 
                                           tf.reduce_sum(loss_wh), 
                                           tf.reduce_sum(loss_conf), 
                                           tf.reduce_sum(loss_class)],  message='loss xy, wh, conf, class: \t',   summarize=1000)   

How do I get these logs 
from the on_batch_end function in the callbacks during training?

thanks

**Comment by kabrau** [comment]
I think that you do not need this, In my version, I commented this lines.
But, if you need, you can save in a txt file or send to tensorboard

**Comment by dndmania** [comment]
Is there any way to get the logs from the callback class in this version? 
T.T

**Comment by kabrau** [comment]
I think not, you need to implement this
